src/SDKs/ServiceFabric/Management.ServiceFabric/Microsoft.Azure.Management.ServiceFabric.csproj

```diff
@@ -7,7 +7,7 @@
 		<PackageId>Microsoft.Azure.Management.ServiceFabric</PackageId>
 		<Description>Microsoft Azure Management ServiceFabric Library</Description>
 		<AssemblyName>Microsoft.Azure.Management.ServiceFabric</AssemblyName>
-		<Version>1.0.1-preview</Version>
+		<Version>1.1.0</Version>
 		<PackageTags>Microsoft Azure Management Library;ServiceFabric;ServiceFabric management;</PackageTags>
 		<PackageReleaseNotes/>
```

Comment:
We highly recommend adding `PackageReleaseNotes` since it helps developers using nuget
You can find an example [here](https://github.com/Azure/azure-sdk-for-net/blob/psSdkJson6/src/SDKs/HDInsight/Management.HDInsight/Management.HDInsight.csproj#L14)

Reply:
@dsgouda Added PackageReleaseNotes section.